src/clients/patreon/tests/PatreonAgent.test.ts

```diff
@@ -133,19 +115,40 @@ describe('PatreonAgent', () => {
         '123456789',
       );
 
-      expect(mockUpdateCurrentPatreonMember).toHaveBeenCalledWith(
+      expect(mockUpdateCurrentPaidPatreonMember).toHaveBeenCalledWith(
         null,
         paidPatronFullDetails,
         'usernamelow',
         patreonUserTokens,
       );
     });
 
+    it('Does not link an unpaid Patreon account to a user', async () => {
+      mockPatreonController.getPatreonUserTokens.mockResolvedValueOnce(
+        patreonUserTokens,
+      );
+      mockPatreonController.getPaidPatronFullDetails.mockResolvedValueOnce(
+        null,
+      );
+
+      await expect(
+        patreonAgent.linkUserToPatreon('usernamelow', 'codeAbc'),
```

Comment:
Can you try `expect(await patreonAgent...)`

Reply:
will throw an error :(